praw/models/reddit/submission.py

```diff
@@ -152,6 +159,11 @@ def flair(self, text: str = "", css_class: str = ""):
             "text": text,
         }
         url = API_PATH["flair"].format(subreddit=self.thing.subreddit)
+        if flair_template_id is not None:
```

Comment:
Is a caller of the method allowed to provide all three arguments? My intuition is that they aren't. So, there should be logic that throws an exception if an invalid combination is given.

Reply:
They should be able to change the text, not sure about the CSS class though.